fixd.js

```diff
@@ -1,14 +1,30 @@
-const fs = require('fs')
-const path = require('path')
-const fixd = require('./src/index')
+'use strict'
+const merge = require('lodash.merge')
 
-// Auto-load built-in plugins
-try {
-  fs.readdirSync(path.resolve(__dirname, './src/plugins')).forEach((f) => {
-    fixd.use(f.replace('.js', ''))
-  })
-} catch (e) {
-  throw new Error('Error loading built-in plugins', e.message)
+const fixd = {
+  /**
+   * Add a fixture to the fixd library and freezes if not in namespace
+   * @param {String} name A string name to represent the fixture
+   * @param {*} value The fixture value
+   */
+  add: (name, val) => {
+    if (Object.keys(fixd).indexOf(name) >= 0) {
+      throw new Error(`Cannot add fixture to already reserved namespace '${name}'.`)
+    }
+    fixd[name] = Object.freeze(val)
+  },
+  /**
+   * Allows mutation of already existing fixture by providing name of fixture and
+   * modifier method
+   * @param {String} name The namespace of the fixture to mutate
+   * @param {Function} modifier A function to modify the fixture
+   */
+  mutate: (name, modifier) => {
```

Comment:
Talked with @levithomason about this, I hate to bikeshed but I think given the API structure this method name is misleading. Because values are added to the store w/ a method call, and accessed with a getter, it needs to be even _more_ clear that `mutate` acts as a getter and not as something that acts on fixd's internal store. The language of `mutate` implies that it will modify what's in the global store.

```js
fixd.add('thing', {})          // adds 'thing' to fixd's store
fixd.thing                     // returns 'thing'
fixd.mutate('thing', /* .. */) // returns 'thing' w/ modifications
```

I think something that is more clear about it being a return value would benefit API sanity. One idea would be `fixed.withMutations()` since this implies that it acts more like a getter.